### PR TITLE
Cloud SOAR Delivery 2 - GA

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) moves content from the following beta article into the main Cloud SOAR articles for the GA release:
https://help.sumologic.com/docs/cloud-soar/cloud-soar-delivery-2/

It also adds a release note here:
https://help.sumologic.com/release-notes-csoar/

Issue number: Asana ticket [Cloud SOAR Delivery 2 - GA](https://app.asana.com/0/1183741177809740/1204894486636149/f)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce.

**Comment by jpipkin1** [comment]
Moved this PR to draft. The term "Delivery 2" might change to a version number instead.

File: docs/cse/automation-service/automation-service-bridge.md [hunk 1]
```diff
@@ -92,6 +92,9 @@ Login to Sumo Logic and create a new [installation token](/docs/manage/security/
    1. Click the **Configuration** button (gear icon) at the top of the Cloud SIEM UI.
    1. Under **Integrations**, select **Automation**.
    1. At the top of the screen, click **Manage Playbooks**.
+      :::note
+      {@import ../../reuse/cse-launch-csoar-automation.md} 
+      :::
```
Comment (jpipkin1) [inline docs/cse/automation-service/automation-service-bridge.md:97]:
```suggestion
```

File: docs/cse/automation-service/automation-service-integrations.md [hunk 2]
```diff
@@ -26,6 +26,9 @@ The following procedure describes how to view integrations already installed to
 1. Click the **Configuration** button (gear icon) at the top of the UI.
 1. Under **Integrations**, select **Automation**.
 1. From the **Automation** screen, click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/>
+   :::note
+   {@import ../../reuse/cse-launch-csoar-automation.md}     
+   :::
```
Comment (jpipkin1) [inline docs/cse/automation-service/automation-service-integrations.md:31]:
```suggestion
```

File: docs/cse/automation-service/automation-service-bridge.md [hunk 3]
```diff
@@ -107,6 +107,9 @@ Login to Sumo Logic and create a new [installation token](/docs/manage/security/
    1. Click the **Configuration** button (gear icon) at the top of the Cloud SIEM UI.
    1. Under **Integrations**, select **Automation**.
    1. At the top of the screen, click **Manage Playbooks**.
+      :::note
+      {@import ../../reuse/cse-launch-csoar-automation.md}
+      :::
```
Comment (jpipkin1) [inline docs/cse/automation-service/automation-service-bridge.md:112]:
```suggestion
```

File: docs/cse/automation-service/automation-service-playbooks.md [hunk 4]
```diff
@@ -24,6 +24,9 @@ The following procedure describes how to view playbooks already installed in you
 1. Click the **Configuration** button (gear icon) at the top of the UI.
 1. Under **Integrations**, select **Automation**.
 1. From the **Automation** screen, click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/>
+   :::note
+   {@import ../../reuse/cse-launch-csoar-automation.md}     
+   :::
```
Comment (jpipkin1) [inline docs/cse/automation-service/automation-service-playbooks.md:29]:
```suggestion
```

File: docs/cse/automation-service/automation-service-playbooks.md [hunk 5]
```diff
@@ -37,6 +37,9 @@ The following procedure provides a brief introduction to how to create a playboo
 1. Click the **Configuration** button (gear icon) at the top of the Cloud SIEM UI.
 1. Under **Integrations**, select **Automation**.
 1. Click **Manage Playbooks**. Previous-created playbooks will display. <br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/>
+   :::note
+   {@import ../../reuse/cse-launch-csoar-automation.md}    
+   :::
```
Comment (jpipkin1) [inline docs/cse/automation-service/automation-service-playbooks.md:42]:
```suggestion
```

File: docs/cse/automation-service/automation-service-examples.md [hunk 6]
```diff
@@ -282,7 +290,10 @@ The following example pulls together elements of the [Simple example](#simple-ex
 1. Edit the VirusTotal OIF resource:
    1. Click the **Configuration** button (gear icon) at the top of the UI.
    1. Under **Integrations**, select **Automation**.
-   1. From the Automation screen, click **Manage Playbooks**.
+   1. From the Automation screen, click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/>
+      :::note
+      {@import ../../reuse/cse-launch-csoar-automation.md}
+      :::
```
Comment (jpipkin1) [inline docs/cse/automation-service/automation-service-examples.md:296]:
```suggestion
   1. From the Automation screen, click **Manage Playbooks**.
```

File: docs/cse/automation-service/automation-service-examples.md [hunk 7]
```diff
@@ -172,7 +177,10 @@ The action uses [IP Quality Score](https://www.ipqualityscore.com/) to gather IP
 1. Create a new IP Quality Score integration:
     1. Click the **Configuration** button (gear icon) at the top of the UI.
     1. Under **Integrations**, select **Automation**.
-    1. From the Automation screen, click **Manage Playbooks**.
+    1. From the Automation screen, click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/>
+      :::note
+      {@import ../../reuse/cse-launch-csoar-automation.md} 
+      :::
```
Comment (jpipkin1) [inline docs/cse/automation-service/automation-service-examples.md:183]:
```suggestion
    1. From the Automation screen, click **Manage Playbooks**.
```

File: docs/cse/automation-service/automation-service-examples.md [hunk 8]
```diff
@@ -94,7 +97,9 @@ The following example shows how to configure a notification that sends an email
 1. Edit the Sumo Logic CIP resource:
    1. Click the **Configuration** button (gear icon) at the top of the UI.
    1. Under **Integrations**, select **Automation**.
-   1. From the Automation screen, click **Manage Playbooks**.
+   1. From the Automation screen, click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/>
+    :::note
+    {@import ../../reuse/cse-launch-csoar-automation.md}
```
Comment (jpipkin1) [inline docs/cse/automation-service/automation-service-examples.md:102]:
```suggestion
   1. From the Automation screen, click **Manage Playbooks**.
```

File: docs/cse/automation-service/automation-service-examples.md [hunk 9]
```diff
@@ -20,7 +20,10 @@ The following example shows how to add an enrichment to an Insight using the “
 1. Edit the VirusTotal OIF resource:
    1. Click the **Configuration** button (gear icon) at the top of the UI.
    1. Under **Integrations**, select **Automation**.
-   1. From the Automation screen, click **Manage Playbooks**.
+   1. From the Automation screen, click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/>
+    :::note
+    {@import ../../reuse/cse-launch-csoar-automation.md}    
+    :::
```
Comment (jpipkin1) [inline docs/cse/automation-service/automation-service-examples.md:26]:
```suggestion
   1. From the Automation screen, click **Manage Playbooks**.
```

File: docs/cse/automation-service/about-automation-service.md [hunk 10]
```diff
@@ -171,7 +176,10 @@ To interact with most of the Automation Service features, you must have at least
 To use [integrations](/docs/cse/automation-service/automation-service-integrations), you must configure the connection for their resources.
 1. Click the **Configuration** button (gear icon) at the top of the Cloud SIEM UI.
 1. Under **Integrations**, select **Automation**.<br/><img src={useBaseUrl('img/cse/automations-config-menu.png')} alt="Automation menu option" width="150"/>
-1. Click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/> 
+1. Click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/>
+:::note
+{@import ../../reuse/cse-launch-csoar-automation.md}  
+:::
```
Comment (jpipkin1) [inline docs/cse/automation-service/about-automation-service.md:182]:
```suggestion
1. Click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/>
```

File: docs/cse/automation-service/about-automation-service.md [hunk 11]
```diff
@@ -171,7 +176,7 @@ To interact with most of the Automation Service features, you must have at least
 To use [integrations](/docs/cse/automation-service/automation-service-integrations), you must configure the connection for their resources.
 1. Click the **Configuration** button (gear icon) at the top of the Cloud SIEM UI.
 1. Under **Integrations**, select **Automation**.<br/><img src={useBaseUrl('img/cse/automations-config-menu.png')} alt="Automation menu option" width="150"/>
-1. Click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/> 
+1. Click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/>
```
Comment (jpipkin1) [inline docs/cse/automation-service/about-automation-service.md:179]:
```suggestion
1. Click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/> 
```

**Comment by jpipkin1** [comment]
We'll need to publish this PR at the same time:
https://github.com/SumoLogic/sumologic-documentation/pull/3361

File: docs/cloud-soar/automation.md [hunk 12]
```diff
@@ -382,14 +381,63 @@ Once the user has created this new action, it can easily be added to the existin
 
 See [Integration Framework](#integration-framework) for more details on utilizing the integration framework within Cloud SOAR.
 
-### Configure Slack for Cloud SOAR
+### Integration Builder
+
+{@import ../reuse/csoar-delivery-2.md}
```
Comment (jpipkin1) [inline docs/cloud-soar/automation.md:386]:
```suggestion
```

File: docs/cloud-soar/automation.md [hunk 13]
```diff
@@ -382,14 +381,62 @@ Once the user has created this new action, it can easily be added to the existin
...
+1. Add the resource information:
+   1. Click the **+** button to the left of **Resources**.
+   1. Give the resource a **Label** and enter the connection configuration needed by the resource. What you enter is specific to the integration you're adding the resource for. Each resource's configuration screen may be different, but in most cases, you will need information such as IP addresses, API tokens, usernames, and passwords for the application you're integrating with. <br/><img src={useBaseUrl('img/cloud-soar/delivery-2-add-resource.png')} alt="Example integration" width="400"/>
+   1. Click **Save**. The new integration is complete.<br/><img src={useBaseUrl('img/cloud-soar/delivery-2-completed-integration-2.png')} alt="Example integration" width="600"/>
+1. To test the new action, click on the action, then click **Test Action** in the dialog that displays.<br/><img src={useBaseUrl('img/cloud-soar/delivery-2-test-action.png')} alt="Example integration" width="400"/>
+
+### Cloud or Bridge execution
+
+{@import ../reuse/csoar-delivery-2.md}
```
Comment (jpipkin1) [inline docs/cloud-soar/automation.md:418]:
```suggestion
```

File: docs/cloud-soar/main-menu.md [hunk 14]
```diff
@@ -85,6 +85,21 @@ You can also export Dashboards to an Excel spreadsheet or PDF document to includ
 
 <img src={useBaseUrl('img/cloud-soar/image137.png')} alt="Dashboard Menu" width="800"/>
 
+### Create a dashboard
+
+{@import ../reuse/csoar-delivery-2.md}
```
Comment (jpipkin1) [inline docs/cloud-soar/main-menu.md:90]:
```suggestion
```

File: docs/cloud-soar/main-menu.md [hunk 15]
```diff
@@ -124,6 +138,28 @@ to create the KPI report template.
 KPI Output Sets can also be generated using the Cloud SOAR API. Please see
 the API manual for additional details.
 
+### Create widgets for dashboards or reports
+
+{@import ../reuse/csoar-delivery-2.md}
```
Comment (jpipkin1) [inline docs/cloud-soar/main-menu.md:143]:
```suggestion
```

File: docs/cloud-soar/global-functions-menu.md [hunk 16]
```diff
@@ -429,6 +429,24 @@ Cloud SOAR audit logs and activity can be reviewed under the **Audit Trail** sec
 
 The **Automation Bridge Monitoring** section displays the status of the automation bridges configured.
 
+## Report
+
+{@import ../reuse/csoar-delivery-2.md}
```
Comment (jpipkin1) [inline docs/cloud-soar/global-functions-menu.md:434]:
```suggestion
```

File: docs/cloud-soar/automation.md [hunk 17]
```diff
@@ -500,6 +546,102 @@ Cloud SOAR has a built-in Slack chat feature that lets you exchange messages wit
...
+1. After a recipient chooses one of the available options, the playbook flow will continue and a message will inform the user or the group about the choice made.<br/><img src={useBaseUrl('img/cloud-soar/integration-slack-user-choice-2.png')} alt="Selected user choice" width="600"/>
+
+#### Bidirectional use cases between Slack and incident management
+
+You can manage Slack communication channels directly by creating/editing various incidents within Cloud SOAR. Here are some use cases:
+
+* Creating an incident <br/>When an incident is created, a conversation channel will automatically be created within your Slack workspace, where the channel name will be formed like this: **incident-<incident_id>**. Furthermore, all users (owners, investigators, groups) who are part of the workspace will be added to the channel.<br/><img src={useBaseUrl('img/cloud-soar/created-incident-fs.png')} alt="Created incident" width="800"/><br/><img src={useBaseUrl('img/cloud-soar/created-slack-channel.png')} alt="Created Slack channel" width="800"/>
```
Comment (jpipkin1) [inline docs/cloud-soar/automation.md:640]:
```suggestion
* Creating an incident <br/>When an incident is created, a conversation channel will automatically be created within your Slack workspace, where the channel name will be formed like this: **incident-incident_id**. Furthermore, all users (owners, investigators, groups) who are part of the workspace will be added to the channel.<br/><img src={useBaseUrl('img/cloud-soar/created-incident-fs.png')} alt="Created incident" width="800"/><br/><img src={useBaseUrl('img/cloud-soar/created-slack-channel.png')} alt="Created Slack channel" width="800"/>
```

**Pull request merged**